templates/hp_comware_display_ip_interface.textfsm

```diff
@@ -0,0 +1,19 @@
+Value INTERFACE (\S+)
```

Comment:
```suggestion
Value INTF (\S+)
Value LINE_STATUS (.+)
Value PROTOCOL_STATUS (\S+)
Value List IPADDR (\S+)
Value MTU (\d+)


Start
  ^\S+\s+current\s+state -> Continue.Record
  ^${INTF}\s+current\s+state:\s+${LINE_STATUS}
  ^Line\s+protocol\s+current\s+state:\s+${PROTOCOL_STATUS}
  ^Internet\s+Address\s+is\s+${IPADDR}\s+Primary
  ^Internet\s+Address\s+is\s+${IPADDR}\s+Sub
  ^The\s+Maximum\s+Transmit\s+Unit:\s+${MTU}\s+bytes
  ^\s*$$
  ^. -> Error
```
I don't think I'd worry about a state transition for this template and use the above. I did change the values to align with templates of the same type such as `cisco_ios_show_ip_interface`.

Next you will need to add the other data to account for. For example:
```
  ^input\s+packets
  ^output\s+packets
```

Reply:
ok, added all data and renamed the values :)